lang/en.yml

```
@@ -36,6 +36,7 @@ en:
   SilverStripe\Forms\ConfirmedPasswordField:
     ATLEAST: 'Passwords must be at least {min} characters long.'
     BETWEEN: 'Passwords must be {min} to {max} characters long.'
+    CANBEEMTPYRIGHT: 'If a password is not set then a hidden random password will be automatically generated. The user will need to click the "I''ve lost my password" link to receive a password reset email.'
```

```suggestion
    CAN_BE_EMPTY_RIGHT: 'If a password is not set, a hidden random password will be automatically generated. The user will need to click the "I''ve lost my password" link to receive a password reset email.'
```
What does "can be empty right" mean? Maybe "RANDOM_IF_EMPTY" or similar would be more suitable?

---

Updated

---

That part doesn't matter enough for me to insist on. Happy with the key change.